docs/index.rst

```diff
@@ -1,14 +1,9 @@
 ScanCode.io documentation
 =========================
 
-Welcome to the very start of your ScanCode.io journey! In this documentation
-you’ll find information on:
-
-- An overview of ScanCode.io and ScanPipe
-- Installation instructions
-- Tutorials to get you started
-- Reference documentation about the ScanPipe concepts, Pipelines, Pipes and more
-- How to make technical contributions to the project and the community
+ScanCode.io is a server to script and automate software composition analysis
```

Comment:
Those changes have nothing to do here. There's a need for rebasing/merging the latest main.

Reply:
I am not able to resolve this issue :( Merge and Rebase says Already up to date.
Also there are different versions of same file in the two branches but its not appearing during merge. 
How do I fix it?

Reply:
Maybe a fresh start? Because I've changed just a file or two.

Reply:
@tdruez I think I messed up a bit while updating branch

Reply:
@AvishrantsSh create a new branch based on latest `main` and commit your new files there.